mlrun/feature_store/steps.py

```diff
@@ -51,7 +80,7 @@ def __init__(self, featureset=None, columns=None, name=None, **kwargs):
                 feature.validator.set_feature(feature)
                 self._validators[key] = feature.validator
 
-    def do(self, event):
```

Comment:
shouldn't it be `do` who will call `_do_storey` and `_do_pandas` depends on the usage? how come it got deleted?

Reply:
all the mlrun steps inheriting class called MLRunStep that implement the `do` method that decide accordung to the type of first coming event if `do=:_do_pandas or do=:_do_storey`